Robot/InfiniteRecharge/src/main/java/frc/robot/commands/SpinMotorsRPM.java

```diff
@@ -0,0 +1,51 @@
+package frc.robot.commands;
+
+import frc.robot.Robot;
+import frc.robot.RobotContainer;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+import java.util.logging.Logger;
+
+/**
+ * An example command that uses an example subsystem.
+ */
+public class SpinMotorsRPM extends CommandBase {
+  @SuppressWarnings({ "PMD.UnusedPrivateField", "PMD.SingularField" })
+
+  private final Logger logger = Logger.getLogger(this.getClass().getName());
+
+  public SpinMotorsRPM() {
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(RobotContainer.getInstance().shooter);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    logger.info("got to Shooter motor Activate");
+    // TODO: Retrieve the desired motor speed
+    RobotContainer.getInstance().shooter.setVelocity(RobotContainer.getInstance().Dashboard.getShooterSetSpeed());
+    NetworkTableInstance.getDefault().getTable("limelight").getEntry("ledMode").forceSetNumber(3);
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    // TODO: Retrieve the desired motor speed
+    RobotContainer.getInstance().shooter.setVelocity(RobotContainer.getInstance().Dashboard.getShooterSetSpeed());
+  }
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {
+    logger.info("got to Shooter Motor Spin Stop");
+    RobotContainer.getInstance().shooter.Stop();
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.